src/analyzer.ts

```diff
@@ -261,7 +273,7 @@ export class Analyzer {
       warnings.push({
         code: 'could-not-load',
         message: `Unable to load import: ${error.message || error}`,
-        sourceRange: scannedImport.sourceRange,
+        sourceRange: scannedImport.urlSourceRange || scannedImport.sourceRange,
```

Comment:
Grabbing one source range or the other? Or generating the entire warning given an import error?